### PR TITLE
Add docker file

### PR DESCRIPTION
**Pull request by mhdfadlyhasan**

### CODE REVIEW

**Review by ArinoJenynof — APPROVED** [review]
LGTM (idk gak pernah pake docker)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:16
+WORKDIR /app
+COPY ../package.json package.json
```
Comment (fadhlanmr) [inline Dockerfile]: kalo mau di simplify pake ini
`COPY package*.json ./
`

Comment (fadhlanmr) [reply Dockerfile]: btw discord ga expose port ya?

**Review by fadhlanmr — APPROVED** [review]
banh kenapa ga `node:lts ` banh

**Pull request merged**